### PR TITLE
Allow 6, 8 Items per Row in CategoriesHighlights

### PR DESCRIPTION
**Pull request by sahanljc**
#### What is the purpose of this pull request?
We can use the `CategoriesHighlights` component to show 2, 4, 6 or 8 items per row. Using the CSS overrides, we can change the card dimensions to make a unique experience in our store.

#### What problem is this solving?
We want to show 6 or 8 items per row in a highlighted categories section. This number should be configurable via the admin CMS storefront.

#### How should this be manually tested?
1. Set the `Quantity of Items` to 6 or 8 and add names and images.
2. Use CSS overrides in `vtex.store-theme` to adjust heights and widths of the squared/rectangular cards and images.

#### Screenshots or example usage
6-items-per-row-ch

#### Types of changes
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/__tests__/components/__snapshots__/Greeting.test.js.snap [hunk 1]
```diff
@@ -19,24 +19,4 @@ exports[`<Greeting /> component should match snapshot with loading 1`] = `
...
-    </span>
-    <span
-      class="firstName pl2 b"
-    >
-      name
-    </span>
-  </div>
-</DocumentFragment>
-`;
+exports[`<Greeting /> component should match snapshot without loading 1`] = `<DocumentFragment />`;
```
Comment (klzns) [inline react/__tests__/components/__snapshots__/Greeting.test.js.snap]: Do you know why that happened? 🤔

Comment (klzns) [reply react/__tests__/components/__snapshots__/Greeting.test.js.snap]: Just checked, it's like that in master.

**Comment by klzns** [comment]
Our tests are not passing, I'll fix them.

So you can rebase your branch with passing tests.

File: react/__tests__/components/__snapshots__/Greeting.test.js.snap (hunk 1 above)
Comment (sahanljc) [reply react/__tests__/components/__snapshots__/Greeting.test.js.snap]: You mean, it fails in the master, right?

**Comment by sahanljc** [comment]
> Remember to update the docs [here](https://github.com/vtex-apps/store-components/blob/098028b8ef855f83fa48b56add13240171f1f707/react/components/CategoriesHighlights/README.md). Thanks!

Done! :+1:

**Comment by victorhmp** [comment]
Closing this due to a lack of changes in a long time.
@sahanljc if you still plan on working in this, feel free to reopen it.

**Pull request closed without merging**